src/main/scala/tilelink/SRAM.scala

```diff
@@ -223,7 +234,7 @@ object TLRAM
     devName: Option[String] = None,
   )(implicit p: Parameters): TLInwardNode =
   {
-    val ram = LazyModule(new TLRAM(address, cacheable, executable, atomics, beatBytes, ecc, devName))
+    val ram = LazyModule(new TLRAM(address, parentLogicalTreeNode, cacheable, executable, atomics, beatBytes, ecc, devName))
```

Comment:
Could you give TLRAM its own `logicalTreeNode`, and add that to parent here instead of having to pass the parent all that way in?

Reply:
The parent is passed to the child starting from the root.